apps/cowswap-frontend/src/common/updaters/orders/ExpiredOrdersUpdater.ts

```diff
@@ -33,26 +32,24 @@ export function ExpiredOrdersUpdater(): null {
       try {
         isUpdating.current = true
 
-        // Filter orders:
+        // Filter expired orders:
+        // - Only eth-flow orders
         // - Owned by the current connected account
-        // - Created in the last 5 min, no further
         // - Not yet refunded
-        const pending = expiredRef.current.filter(({ owner, creationTime: creationTimeString, refundHash }) => {
-          const creationTime = new Date(creationTimeString).getTime()
+        const orderWithoutRefund = expiredRef.current.filter(({ owner, refundHash, sellToken }) => {
+          const isEthFlowOrder = sellToken === NATIVE_CURRENCY_BUY_ADDRESS
```

Comment:
Didn't you say there was an util for this?

Reply:
Yes, but it is located in `modules/swap/containers/EthFlowStepper/index.tsx` and if I move it somewhere else it will cause lots of changes. I prefer to refactor that part in another PR